### ISSUE TITLE
Fehler „Request failed with status code 404“ 

### ISSUE THREAD

**Issue by spirit0607** [TA2k/ioBroker.vaillant#30]
Hallo,

wenn ich versuche unter ManuelModeSetpoint eine Temperatur einzustellen, bekomme ich im Protokoll eine Fehlermeldung „Request failed with status code 404“ 

Ich habe eine Vaillant Wärmepumpe Arotherm Split 75/6 mit dem Modul VRC 720 sensocomfort

![7C67BBF9-7871-479E-9746-F6CA6F331DBC](https://user-images.githubusercontent.com/69988146/225102542-6612f0b8-fb30-4d8f-a44e-09d2290f7b12.jpeg)
![5FCCEBAC-7F57-4837-B069-A3C95E6463E1](https://user-images.githubusercontent.com/69988146/225102547-769fb6b9-b0b9-4be7-bc7d-5b1f3c0c9386.jpeg)

 - Adapter version: 0.1.1. von Github

Ich hoffe, dass mir jemand weiterhelfen kann. Vielen Dank jedenfalls schonmal dafür!

**Comment by TA2k** [comment]
Bitte al im forum nachfragen
https://forum.iobroker.net/topic/29221/test-adapter-vaillant-multimatic-senso-und-myvaillant/2?_=1678826778173

**Comment by rvandiepen** [comment]
> Hallo,
> 
> wenn ich versuche unter ManuelModeSetpoint eine Temperatur einzustellen, bekomme ich im Protokoll eine Fehlermeldung „Request failed with status code 404“
> 
> Ich habe eine Vaillant Wärmepumpe Arotherm Split 75/6 mit dem Modul VRC 720 sensocomfort
> 
> ![7C67BBF9-7871-479E-9746-F6CA6F331DBC](https://user-images.githubusercontent.com/69988146/225102542-6612f0b8-fb30-4d8f-a44e-09d2290f7b12.jpeg) ![5FCCEBAC-7F57-4837-B069-A3C95E6463E1](https://user-images.githubusercontent.com/69988146/225102547-769fb6b9-b0b9-4be7-bc7d-5b1f3c0c9386.jpeg)
> 
> * Adapter version: 0.1.1. von Github
> 
> Ich hoffe, dass mir jemand weiterhelfen kann. Vielen Dank jedenfalls schonmal dafür!

Ich hatte das selbste problem.
In die main.js datei sind referenzen nach https://identity.vaillant-group.com/auth/realms/vaillant-germany-b2c
Ich wohne in Belgien und meine konto nutzt https://identity.vaillant-group.com/auth/realms/vaillant-belgium-b2c
Ich habe das geandert in die main.js datei und mittweil klapt es wieder.

**Comment by chrispi68** [comment]
Ich habe seit gestern ebenfalls eine Wärmepumpe und das gleiche Problem. Wohne in Deutschland. Eingesetzt wird MyVaillant.

```
vaillant.0 | 2023-04-19 18:07:39.471 | error | {"timestamp":"2023-04-19T18:07:39.457+00:00","status":404,"error":"Not Found","path":"/systems/c0ed66c9-86d2-463d-b730-c197926ba16a/zones/0/manualModeSetpoint"}

vaillant.0 | 2023-04-19 18:07:39.471 | error | AxiosError: Request failed with status code 404
```

Ich habe zwei Zonen. Die Fehlermeldung tritt nur beim Setzen von manualModeSetpoint in Zone 1 auf. Beim Setzen von Zone 2 erscheint kein Fehler, es wird aber ebenfalls kleine Temperatur verändert. Der Aufruf bleibt also ohne Wirkung.

Vorher hatte ich eine Gasheizung von Vaillant mit einer einzelnen Zone und ebenfalls MyVaillant. Wir gäbe es keine Probleme.

**Comment by chrispi68** [comment]
Ich habe weiter geprüft: Es funktionieren generell keine Schreibzugriffe auf alle Werte der Zonen 1 und 2. Es tauchen keine Fehler im Log auf. Ausnahme ist nur manualModeSetpoint. Hier taucht der oben beschriebene Fehler auf.

<img width="2092" alt="Bildschirmfoto 2023-04-20 um 12 28 25" src="https://user-images.githubusercontent.com/33786075/233339806-f91291f5-2ccf-435f-a898-2c87c6ba94ec.png">

**Comment by smartboart** [comment]
seleb konstellation selbes problem auch seid der Umstellung auf my vaillant app..

`<html><body>


vaillant.0 | 2023-06-02 21:30:56.809 | error | {"statusCode":404,"message":"Resource not found"}
-- | -- | -- | --
vaillant.0 | 2023-06-02 21:30:56.809 | error | AxiosError: Request failed with status code 404
vaillant.0 | 2023-06-02 21:27:21.587 | error | {"timestamp":"2023-06-02T19:27:21.575+00:00","status":404,"error":"Not   Found","path":"/systems/2c5eff7b-53ea-4d80-a823-910753b37555/zones/0/manualModeSetpoint"}
vaillant.0 | 2023-06-02 21:27:21.586 | error | AxiosError: Request failed with status code 404
vaillant.0 | 2023-06-02 21:26:29.715 | error | {"timestamp":"2023-06-02T19:26:29.692+00:00","status":404,"error":"Not   Found","path":"/systems/2c5eff7b-53ea-4d80-a823-910753b37555/zones/0/manualModeSetpoint"}


</body>
</html>`

**Comment by Vocta1310** [comment]
Selbes Problem hier - seit 28.08.2023 meldet das Protokoll:

{"timestamp":"2023-08-30T07:40:54.629+00:00","status":404,"error":"Not Found","path":"/systems"}
AxiosError: Request failed with status code 404

Realm: germany - App-seitig wurde nichts geändert

Die anderen realms habe ich auch mal durchgetestet - immer mit dem Fehler, dass das Login nicht passt. Sollte also daran nicht liegen.

**Comment by TA2k** [comment]
Bitte die GitHub version testen

**Comment by chrispi68** [comment]
> Bitte die GitHub version testen

Vielen Dank, jetzt funktioniert es wieder.

**Comment by chrispi68** [comment]
Mir ist gerade aufgefallen, dass das Objekt "systemControlState" fehlt.

<img width="1275" alt="Bildschirmfoto 2023-08-31 um 13 04 44" src="https://github.com/TA2k/ioBroker.vaillant/assets/33786075/7c6e95d0-8c26-4e20-bdc1-ceb88fadca3f">

**Comment by TA2k** [comment]
Ja es bedarf einer größeren Änderung

**Comment by spirit0607** [comment]
> Mir ist gerade aufgefallen, dass der Ordner "vaillant/0/c0ed66c9-86d2-463d-b730-c197926ba16a/systemControlState" fehlt.
> 
> 
> 
> <img width="1275" alt="Bildschirmfoto 2023-08-31 um 13 04 44" src="https://github.com/TA2k/ioBroker.vaillant/assets/33786075/7c6e95d0-8c26-4e20-bdc1-ceb88fadca3f">
> 
> 

Ja, bei mir genau das gleiche Problem. Leider kann ich die Anlage seit einiger Zeit nicht mehr über iobroker steuern.

**Comment by spirit0607** [comment]
Ja, mit v 0.20 von Github

**Comment by TA2k** [comment]
Sollte eigentlich ein state ordner geben. Was sagt der log

**Comment by spirit0607** [comment]
Einen state Ordner habe ich. Jedoch kann ich da ja keine Steuerung drüber machen. Das ging immer im Ordner "systemcontrol" und der fehlt bei mir.

**Comment by spirit0607** [comment]
Gibt es hier schon eine Lösung? Ich sehe leider noch immer nicht den Ordner "systemControlState" Habe die Version 0.20 von Github.

**Comment by TA2k** [comment]
Bisher noch nicht nur die standard remote Sachen sind bisher drin

**Comment by spirit0607** [comment]
Okay, danke für die schnelle Rückmeldung. Was heißt das denn genau? Ich finde bei mir irgendwie nichts zum einstellen. Ich wollte die Warmwassertemperatur steuern. Das ging früher, jetzt aber nicht mehr.

**Comment by TA2k** [comment]
bitte GitHub version testen unter remote

**Comment by spirit0607** [comment]
<img width="1849" alt="Bildschirmfoto 2023-10-04 um 19 35 42" src="https://github.com/TA2k/ioBroker.vaillant/assets/69988146/81dcd22c-d254-4b0e-9db2-a54ce988075c">

<img width="789" alt="Bildschirmfoto 2023-10-04 um 19 37 31" src="https://github.com/TA2k/ioBroker.vaillant/assets/69988146/407952d9-42fb-4bdb-a9bb-238e62acee1c">

Unter remote finde ich nichts. So siehts bei mir aus.

**Comment by TA2k** [comment]
hast du via GitHub installiert und ein MyVaillant account

**Comment by spirit0607** [comment]
Ja genau. Hab den Expertenmodus in iobroker angemacht. Dann auf "die Katze" geklickt und über GitHub nach Vaillant gesucht. Und diese Version installiert. Nutze auch myVaillant
